features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/AsyncErrorLaunchScenario.kt

```diff
@@ -10,8 +10,12 @@ import com.bugsnag.android.*
 internal class AsyncErrorLaunchScenario(config: Configuration,
                                         context: Context) : Scenario(config, context) {
 
-    override fun run() {
+    init {
         config.delivery = createSlowDelivery(context)
+        config.setAutoCaptureSessions(false)
```

Comment:
True, though it was hard to tell at a glance which tests assumed an otherwise true-by-default value was false, and for which ones it was true, and then impossible to test the what the default result is for a new Configuration without alteration. 

Sometimes reducing duplication increases the overall complexity of adding new tests (or understanding existing ones) at the same time.